<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title>测试报告</title>
    <meta name="generator" content="HTMLTestRunner 0.8.2.2"/>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    <link href="http://libs.baidu.com/bootstrap/3.0.3/css/bootstrap.min.css" rel="stylesheet">
    <script src="http://libs.baidu.com/jquery/2.0.0/jquery.min.js"></script>
    <script src="http://libs.baidu.com/bootstrap/3.0.3/js/bootstrap.min.js"></script>
    
<style type="text/css" media="screen">
body        { font-family: Microsoft YaHei,Tahoma,arial,helvetica,sans-serif;padding: 20px; font-size: 120%; }
table       { font-size: 100%; }
/* -- heading ---------------------------------------------------------------------- */
.heading {
    margin-top: 0ex;
    margin-bottom: 1ex;
}
.heading .description {
    margin-top: 4ex;
    margin-bottom: 6ex;
}
/* -- report ------------------------------------------------------------------------ */
#total_row  { font-weight: bold; }
.passCase   { color: #5cb85c; }
.failCase   { color: #d9534f; font-weight: bold; }
.errorCase  { color: #f0ad4e; font-weight: bold; }
.hiddenRow  { display: none; }
.testcase   { margin-left: 2em; }
</style>

</head>
<body >
<script language="javascript" type="text/javascript">
output_list = Array();
/*level 调整增加只显示通过用例的分类 --Findyou
0:Summary //all hiddenRow
1:Failed  //pt hiddenRow, ft none
2:Pass    //pt none, ft hiddenRow
3:All     //pt none, ft none
*/
function showCase(level) {
    trs = document.getElementsByTagName("tr");
    for (var i = 0; i < trs.length; i++) {
        tr = trs[i];
        id = tr.id;
        if (id.substr(0,2) == 'ft') {
            if (level == 2 || level == 0 ) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
        if (id.substr(0,2) == 'pt') {
            if (level < 2) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
    }
    //加入【详细】切换文字变化 --Findyou
    detail_class=document.getElementsByClassName('detail');
	//console.log(detail_class.length)
	if (level == 3) {
		for (var i = 0; i < detail_class.length; i++){
			detail_class[i].innerHTML="收起"
		}
	}
	else{
			for (var i = 0; i < detail_class.length; i++){
			detail_class[i].innerHTML="详细"
		}
	}
}
function showClassDetail(cid, count) {
    var id_list = Array(count);
    var toHide = 1;
    for (var i = 0; i < count; i++) {
        //ID修改 点 为 下划线 -Findyou
        tid0 = 't' + cid.substr(1) + '_' + (i+1);
        tid = 'f' + tid0;
        tr = document.getElementById(tid);
        if (!tr) {
            tid = 'p' + tid0;
            tr = document.getElementById(tid);
        }
        id_list[i] = tid;
        if (tr.className) {
            toHide = 0;
        }
    }
    for (var i = 0; i < count; i++) {
        tid = id_list[i];
        //修改点击无法收起的BUG，加入【详细】切换文字变化 --Findyou
        if (toHide) {
            document.getElementById(tid).className = 'hiddenRow';
            document.getElementById(cid).innerText = "详细"
        }
        else {
            document.getElementById(tid).className = '';
            document.getElementById(cid).innerText = "收起"
        }
    }
}
function html_escape(s) {
    s = s.replace(/&/g,'&amp;');
    s = s.replace(/</g,'&lt;');
    s = s.replace(/>/g,'&gt;');
    return s;
}
</script>
<div class='heading'>
<h1 style="font-family: Microsoft YaHei">测试报告</h1>
<p class='attribute'><strong>测试人员 : </strong> JM</p>
<p class='attribute'><strong>开始时间 : </strong> 2020-03-05 18:51:43</p>
<p class='attribute'><strong>合计耗时 : </strong> 0:00:04.081000</p>
<p class='attribute'><strong>测试结果 : </strong> 共 19，通过 1，错误 18，通过率= 5.26%</p>

<p class='description'>测试用</p>
</div>


<p id='show_detail_line'>
<a class="btn btn-primary" href='javascript:showCase(0)'>概要{ 5.26% }</a>
<a class="btn btn-danger" href='javascript:showCase(1)'>失败{ 0 }</a>
<a class="btn btn-success" href='javascript:showCase(2)'>通过{ 1 }</a>
<a class="btn btn-info" href='javascript:showCase(3)'>所有{ 19 }</a>
</p>
<table id='result_table' class="table table-condensed table-bordered table-hover">
<colgroup>
<col align='left' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
</colgroup>
<tr id='header_row' class="text-center success" style="font-weight: bold;font-size: 16px;">
    <td>用例集/测试用例</td>
    <td>总计</td>
    <td>通过</td>
    <td>失败</td>
    <td>错误</td>
    <td>详细</td>
</tr>

<tr class='errorClass warning'>
    <td>test_case.test_case_sit.TestHttp01</td>
    <td class="text-center">19</td>
    <td class="text-center">1</td>
    <td class="text-center">0</td>
    <td class="text-center">18</td>
    <td class="text-center"><a href="javascript:showClassDetail('c1',19)" class="detail" id='c1'>详细</a></td>
</tr>

<tr id='ft1_1' class='none'>
    <td class='errorCase'><div class='testcase'>test_001_01</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_1'>错误</button>
    <div id='div_ft1_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_1'>错误</button>
    <div id='div_ft1_1' class="collapse in" align="left">
    <pre>
    
ft1_1: ----------------------------------------
https://mobileuat.utcook.com//auth-internet/resource/getResourceTags
测试场景：获取指定资源的标签列表_正常获取[200]
调用的地址是:https://mobileuat.utcook.com//auth-internet/resource/getResourceTags?authSpace=key_authSpace200_001&amp;resourceKey=ResourceKey200_001
请求参数:{'authSpace': 'key_authSpace200_001', 'resourceKey': 'ResourceKey200_001'}
接口调用成功: &lt;Response [200]&gt;
----------------------------------------
Traceback (most recent call last):
  File "D:\Program Files\python\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\jiangman\newtest\test_authority\test_case\test_case_sit.py", line 41, in test_001
    do_excel().into_date(num + 1, (do_excel().max_column + 1), res.status_code)
  File "E:\jiangman\newtest\test_authority\common\do_excel.py", line 35, in into_date
    self.wb.save(get_common().get_excel()[0])#保存excel
  File "D:\Program Files\python\lib\site-packages\openpyxl\workbook\workbook.py", line 408, in save
    save_workbook(self, filename)
  File "D:\Program Files\python\lib\site-packages\openpyxl\writer\excel.py", line 291, in save_workbook
    archive = ZipFile(filename, 'w', ZIP_DEFLATED, allowZip64=True)
  File "D:\Program Files\python\lib\zipfile.py", line 1204, in __init__
    self.fp = io.open(file, filemode)
PermissionError: [Errno 13] Permission denied: 'E:\\jiangman\\newtest\\test_authority\\test_data\\word.xlsx'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_2' class='none'>
    <td class='errorCase'><div class='testcase'>test_001_02</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_2'>错误</button>
    <div id='div_ft1_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_2'>错误</button>
    <div id='div_ft1_2' class="collapse in" align="left">
    <pre>
    
ft1_2: ----------------------------------------
https://mobileuat.utcook.com//auth-internet/resource/getResourceTags
测试场景：获取指定资源的标签列表_有效校验_authSpace不存在500
调用的地址是:https://mobileuat.utcook.com//auth-internet/resource/getResourceTags?authSpace=key0_authSpace200_001&amp;resourceKey=ResourceKey200_001
请求参数:{'authSpace': 'key0_authSpace200_001', 'resourceKey': 'ResourceKey200_001'}
接口调用失败:{'utMsg': '开发者[jiangman]不存在此权限空间[key0_authSpace200_001]', 'utCode': 1201, 'utService': 'auth-internet-service'}:
----------------------------------------
Traceback (most recent call last):
  File "D:\Program Files\python\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\jiangman\newtest\test_authority\test_case\test_case_sit.py", line 41, in test_001
    do_excel().into_date(num + 1, (do_excel().max_column + 1), res.status_code)
  File "E:\jiangman\newtest\test_authority\common\do_excel.py", line 35, in into_date
    self.wb.save(get_common().get_excel()[0])#保存excel
  File "D:\Program Files\python\lib\site-packages\openpyxl\workbook\workbook.py", line 408, in save
    save_workbook(self, filename)
  File "D:\Program Files\python\lib\site-packages\openpyxl\writer\excel.py", line 291, in save_workbook
    archive = ZipFile(filename, 'w', ZIP_DEFLATED, allowZip64=True)
  File "D:\Program Files\python\lib\zipfile.py", line 1204, in __init__
    self.fp = io.open(file, filemode)
PermissionError: [Errno 13] Permission denied: 'E:\\jiangman\\newtest\\test_authority\\test_data\\word.xlsx'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_3' class='none'>
    <td class='errorCase'><div class='testcase'>test_001_03</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_3' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_3'>错误</button>
    <div id='div_ft1_3' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_3' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_3'>错误</button>
    <div id='div_ft1_3' class="collapse in" align="left">
    <pre>
    
ft1_3: ----------------------------------------
https://mobileuat.utcook.com//auth-internet/resource/getResourceTags
测试场景：获取指定资源的标签列表_有效校验_resourceKey不存在500
调用的地址是:https://mobileuat.utcook.com//auth-internet/resource/getResourceTags?authSpace=key_authSpace200_001&amp;resourceKey=ResourceKey0_001
请求参数:{'authSpace': 'key_authSpace200_001', 'resourceKey': 'ResourceKey0_001'}
接口调用失败:{'utMsg': '权限空间[key_authSpace200_001]中，不存在该资源[ResourceKey0_001]', 'utCode': 1205, 'utService': 'auth-internet-service'}:
----------------------------------------
Traceback (most recent call last):
  File "D:\Program Files\python\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\jiangman\newtest\test_authority\test_case\test_case_sit.py", line 41, in test_001
    do_excel().into_date(num + 1, (do_excel().max_column + 1), res.status_code)
  File "E:\jiangman\newtest\test_authority\common\do_excel.py", line 35, in into_date
    self.wb.save(get_common().get_excel()[0])#保存excel
  File "D:\Program Files\python\lib\site-packages\openpyxl\workbook\workbook.py", line 408, in save
    save_workbook(self, filename)
  File "D:\Program Files\python\lib\site-packages\openpyxl\writer\excel.py", line 291, in save_workbook
    archive = ZipFile(filename, 'w', ZIP_DEFLATED, allowZip64=True)
  File "D:\Program Files\python\lib\zipfile.py", line 1204, in __init__
    self.fp = io.open(file, filemode)
PermissionError: [Errno 13] Permission denied: 'E:\\jiangman\\newtest\\test_authority\\test_data\\word.xlsx'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_4' class='none'>
    <td class='errorCase'><div class='testcase'>test_001_04</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_4' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_4'>错误</button>
    <div id='div_ft1_4' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_4' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_4'>错误</button>
    <div id='div_ft1_4' class="collapse in" align="left">
    <pre>
    
ft1_4: ----------------------------------------
https://mobileuat.utcook.com//auth-internet/resource/getResourceTags
测试场景：获取指定资源的标签列表_有效校验__其它开发者有效authSpace500
调用的地址是:https://mobileuat.utcook.com//auth-internet/resource/getResourceTags?authSpace=key01_authSpace200_001&amp;resourceKey=ResourceKey200_001
请求参数:{'authSpace': 'key01_authSpace200_001', 'resourceKey': 'ResourceKey200_001'}
接口调用失败:{'utMsg': '开发者[jiangman]不存在此权限空间[key01_authSpace200_001]', 'utCode': 1201, 'utService': 'auth-internet-service'}:
----------------------------------------
Traceback (most recent call last):
  File "D:\Program Files\python\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\jiangman\newtest\test_authority\test_case\test_case_sit.py", line 41, in test_001
    do_excel().into_date(num + 1, (do_excel().max_column + 1), res.status_code)
  File "E:\jiangman\newtest\test_authority\common\do_excel.py", line 35, in into_date
    self.wb.save(get_common().get_excel()[0])#保存excel
  File "D:\Program Files\python\lib\site-packages\openpyxl\workbook\workbook.py", line 408, in save
    save_workbook(self, filename)
  File "D:\Program Files\python\lib\site-packages\openpyxl\writer\excel.py", line 291, in save_workbook
    archive = ZipFile(filename, 'w', ZIP_DEFLATED, allowZip64=True)
  File "D:\Program Files\python\lib\zipfile.py", line 1204, in __init__
    self.fp = io.open(file, filemode)
PermissionError: [Errno 13] Permission denied: 'E:\\jiangman\\newtest\\test_authority\\test_data\\word.xlsx'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_5' class='none'>
    <td class='errorCase'><div class='testcase'>test_001_05</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_5' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_5'>错误</button>
    <div id='div_ft1_5' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_5' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_5'>错误</button>
    <div id='div_ft1_5' class="collapse in" align="left">
    <pre>
    
ft1_5: ----------------------------------------
https://mobileuat.utcook.com//auth-internet/resource/getResourceTags
测试场景：获取指定资源的标签列表_必填校验_authSpace为空500
调用的地址是:https://mobileuat.utcook.com//auth-internet/resource/getResourceTags?authSpace=&amp;resourceKey=ResourceKey200_001
请求参数:{'authSpace': '', 'resourceKey': 'ResourceKey200_001'}
接口调用失败:{'utMsg': 'authSpace不能为空！', 'utCode': 1001, 'utService': 'auth-internet-service'}:
----------------------------------------
Traceback (most recent call last):
  File "D:\Program Files\python\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\jiangman\newtest\test_authority\test_case\test_case_sit.py", line 41, in test_001
    do_excel().into_date(num + 1, (do_excel().max_column + 1), res.status_code)
  File "E:\jiangman\newtest\test_authority\common\do_excel.py", line 35, in into_date
    self.wb.save(get_common().get_excel()[0])#保存excel
  File "D:\Program Files\python\lib\site-packages\openpyxl\workbook\workbook.py", line 408, in save
    save_workbook(self, filename)
  File "D:\Program Files\python\lib\site-packages\openpyxl\writer\excel.py", line 291, in save_workbook
    archive = ZipFile(filename, 'w', ZIP_DEFLATED, allowZip64=True)
  File "D:\Program Files\python\lib\zipfile.py", line 1204, in __init__
    self.fp = io.open(file, filemode)
PermissionError: [Errno 13] Permission denied: 'E:\\jiangman\\newtest\\test_authority\\test_data\\word.xlsx'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_6' class='none'>
    <td class='errorCase'><div class='testcase'>test_001_06</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_6' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_6'>错误</button>
    <div id='div_ft1_6' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_6' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_6'>错误</button>
    <div id='div_ft1_6' class="collapse in" align="left">
    <pre>
    
ft1_6: ----------------------------------------
https://mobileuat.utcook.com//auth-internet/resource/getResourceTags
测试场景：获取指定资源的标签列表_必填校验_resourceKey为空500
调用的地址是:https://mobileuat.utcook.com//auth-internet/resource/getResourceTags?authSpace=key_authSpace200_001&amp;resourceKey=
请求参数:{'authSpace': 'key_authSpace200_001', 'resourceKey': ''}
接口调用失败:{'utMsg': 'resourceKey不能为空！', 'utCode': 1002, 'utService': 'auth-internet-service'}:
----------------------------------------
Traceback (most recent call last):
  File "D:\Program Files\python\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\jiangman\newtest\test_authority\test_case\test_case_sit.py", line 41, in test_001
    do_excel().into_date(num + 1, (do_excel().max_column + 1), res.status_code)
  File "E:\jiangman\newtest\test_authority\common\do_excel.py", line 35, in into_date
    self.wb.save(get_common().get_excel()[0])#保存excel
  File "D:\Program Files\python\lib\site-packages\openpyxl\workbook\workbook.py", line 408, in save
    save_workbook(self, filename)
  File "D:\Program Files\python\lib\site-packages\openpyxl\writer\excel.py", line 291, in save_workbook
    archive = ZipFile(filename, 'w', ZIP_DEFLATED, allowZip64=True)
  File "D:\Program Files\python\lib\zipfile.py", line 1204, in __init__
    self.fp = io.open(file, filemode)
PermissionError: [Errno 13] Permission denied: 'E:\\jiangman\\newtest\\test_authority\\test_data\\word.xlsx'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_7' class='none'>
    <td class='errorCase'><div class='testcase'>test_001_07</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_7' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_7'>错误</button>
    <div id='div_ft1_7' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_7' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_7'>错误</button>
    <div id='div_ft1_7' class="collapse in" align="left">
    <pre>
    
ft1_7: ----------------------------------------
https://mobileuat.utcook.com//auth-internet/resource/getResourceTags
测试场景：获取指定资源的标签列表_格式校验_"authSpace"不包含下划线[200]
调用的地址是:https://mobileuat.utcook.com//auth-internet/resource/getResourceTags?authSpace=authSpace200&amp;resourceKey=ResourceKey200_001
请求参数:{'authSpace': 'authSpace200', 'resourceKey': 'ResourceKey200_001'}
接口调用成功: &lt;Response [200]&gt;
----------------------------------------
Traceback (most recent call last):
  File "D:\Program Files\python\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\jiangman\newtest\test_authority\test_case\test_case_sit.py", line 41, in test_001
    do_excel().into_date(num + 1, (do_excel().max_column + 1), res.status_code)
  File "E:\jiangman\newtest\test_authority\common\do_excel.py", line 35, in into_date
    self.wb.save(get_common().get_excel()[0])#保存excel
  File "D:\Program Files\python\lib\site-packages\openpyxl\workbook\workbook.py", line 408, in save
    save_workbook(self, filename)
  File "D:\Program Files\python\lib\site-packages\openpyxl\writer\excel.py", line 291, in save_workbook
    archive = ZipFile(filename, 'w', ZIP_DEFLATED, allowZip64=True)
  File "D:\Program Files\python\lib\zipfile.py", line 1204, in __init__
    self.fp = io.open(file, filemode)
PermissionError: [Errno 13] Permission denied: 'E:\\jiangman\\newtest\\test_authority\\test_data\\word.xlsx'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_8' class='none'>
    <td class='errorCase'><div class='testcase'>test_001_08</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_8' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_8'>错误</button>
    <div id='div_ft1_8' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_8' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_8'>错误</button>
    <div id='div_ft1_8' class="collapse in" align="left">
    <pre>
    
ft1_8: ----------------------------------------
https://mobileuat.utcook.com//auth-internet/resource/getResourceTags
测试场景：获取指定资源的标签列表_格式校验_"authSpace"不包含小写字母[200]
调用的地址是:https://mobileuat.utcook.com//auth-internet/resource/getResourceTags?authSpace=2000&amp;resourceKey=ResourceKey200_001
请求参数:{'authSpace': '2000', 'resourceKey': 'ResourceKey200_001'}
接口调用成功: &lt;Response [200]&gt;
----------------------------------------
Traceback (most recent call last):
  File "D:\Program Files\python\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\jiangman\newtest\test_authority\test_case\test_case_sit.py", line 41, in test_001
    do_excel().into_date(num + 1, (do_excel().max_column + 1), res.status_code)
  File "E:\jiangman\newtest\test_authority\common\do_excel.py", line 35, in into_date
    self.wb.save(get_common().get_excel()[0])#保存excel
  File "D:\Program Files\python\lib\site-packages\openpyxl\workbook\workbook.py", line 408, in save
    save_workbook(self, filename)
  File "D:\Program Files\python\lib\site-packages\openpyxl\writer\excel.py", line 291, in save_workbook
    archive = ZipFile(filename, 'w', ZIP_DEFLATED, allowZip64=True)
  File "D:\Program Files\python\lib\zipfile.py", line 1204, in __init__
    self.fp = io.open(file, filemode)
PermissionError: [Errno 13] Permission denied: 'E:\\jiangman\\newtest\\test_authority\\test_data\\word.xlsx'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_9' class='none'>
    <td class='errorCase'><div class='testcase'>test_001_09</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_9' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_9'>错误</button>
    <div id='div_ft1_9' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_9' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_9'>错误</button>
    <div id='div_ft1_9' class="collapse in" align="left">
    <pre>
    
ft1_9: ----------------------------------------
https://mobileuat.utcook.com//auth-internet/resource/getResourceTags
测试场景：获取指定资源的标签列表_格式校验_"authSpace"包含特殊字符[500]
调用的地址是:https://mobileuat.utcook.com//auth-internet/resource/getResourceTags?authSpace=001%2A%2A&amp;resourceKey=ResourceKey200_001
请求参数:{'authSpace': '001**', 'resourceKey': 'ResourceKey200_001'}
接口调用失败:{'utMsg': 'authSpace仅支持4-32个字符，允许大小写字母数字下划线！', 'utCode': 1114, 'utService': 'auth-internet-service'}:
----------------------------------------
Traceback (most recent call last):
  File "D:\Program Files\python\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\jiangman\newtest\test_authority\test_case\test_case_sit.py", line 41, in test_001
    do_excel().into_date(num + 1, (do_excel().max_column + 1), res.status_code)
  File "E:\jiangman\newtest\test_authority\common\do_excel.py", line 35, in into_date
    self.wb.save(get_common().get_excel()[0])#保存excel
  File "D:\Program Files\python\lib\site-packages\openpyxl\workbook\workbook.py", line 408, in save
    save_workbook(self, filename)
  File "D:\Program Files\python\lib\site-packages\openpyxl\writer\excel.py", line 291, in save_workbook
    archive = ZipFile(filename, 'w', ZIP_DEFLATED, allowZip64=True)
  File "D:\Program Files\python\lib\zipfile.py", line 1204, in __init__
    self.fp = io.open(file, filemode)
PermissionError: [Errno 13] Permission denied: 'E:\\jiangman\\newtest\\test_authority\\test_data\\word.xlsx'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_10' class='none'>
    <td class='errorCase'><div class='testcase'>test_001_10</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_10' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_10'>错误</button>
    <div id='div_ft1_10' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_10' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_10'>错误</button>
    <div id='div_ft1_10' class="collapse in" align="left">
    <pre>
    
ft1_10: ----------------------------------------
https://mobileuat.utcook.com//auth-internet/resource/getResourceTags
测试场景：获取指定资源的标签列表_格式校验_resourceKey不包含字母[200]
调用的地址是:https://mobileuat.utcook.com//auth-internet/resource/getResourceTags?authSpace=key_authSpace200_001&amp;resourceKey=5000
请求参数:{'authSpace': 'key_authSpace200_001', 'resourceKey': '5000'}
接口调用成功: &lt;Response [200]&gt;
----------------------------------------
Traceback (most recent call last):
  File "D:\Program Files\python\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\jiangman\newtest\test_authority\test_case\test_case_sit.py", line 41, in test_001
    do_excel().into_date(num + 1, (do_excel().max_column + 1), res.status_code)
  File "E:\jiangman\newtest\test_authority\common\do_excel.py", line 35, in into_date
    self.wb.save(get_common().get_excel()[0])#保存excel
  File "D:\Program Files\python\lib\site-packages\openpyxl\workbook\workbook.py", line 408, in save
    save_workbook(self, filename)
  File "D:\Program Files\python\lib\site-packages\openpyxl\writer\excel.py", line 291, in save_workbook
    archive = ZipFile(filename, 'w', ZIP_DEFLATED, allowZip64=True)
  File "D:\Program Files\python\lib\zipfile.py", line 1204, in __init__
    self.fp = io.open(file, filemode)
PermissionError: [Errno 13] Permission denied: 'E:\\jiangman\\newtest\\test_authority\\test_data\\word.xlsx'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_11' class='none'>
    <td class='errorCase'><div class='testcase'>test_001_11</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_11' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_11'>错误</button>
    <div id='div_ft1_11' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_11' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_11'>错误</button>
    <div id='div_ft1_11' class="collapse in" align="left">
    <pre>
    
ft1_11: ----------------------------------------
https://mobileuat.utcook.com//auth-internet/resource/getResourceTags
测试场景：获取指定资源的标签列表_格式校验_resourceKey包含特殊字符[500]
调用的地址是:https://mobileuat.utcook.com//auth-internet/resource/getResourceTags?authSpace=key_authSpace200_001&amp;resourceKey=ResourceKey200_001%21%40%24
请求参数:{'authSpace': 'key_authSpace200_001', 'resourceKey': 'ResourceKey200_001!@$'}
接口调用失败:{'utMsg': 'resourceKey仅支持4-32个字符，允许大小写字母数字下划线！', 'utCode': 1102, 'utService': 'auth-internet-service'}:
----------------------------------------
Traceback (most recent call last):
  File "D:\Program Files\python\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\jiangman\newtest\test_authority\test_case\test_case_sit.py", line 41, in test_001
    do_excel().into_date(num + 1, (do_excel().max_column + 1), res.status_code)
  File "E:\jiangman\newtest\test_authority\common\do_excel.py", line 35, in into_date
    self.wb.save(get_common().get_excel()[0])#保存excel
  File "D:\Program Files\python\lib\site-packages\openpyxl\workbook\workbook.py", line 408, in save
    save_workbook(self, filename)
  File "D:\Program Files\python\lib\site-packages\openpyxl\writer\excel.py", line 291, in save_workbook
    archive = ZipFile(filename, 'w', ZIP_DEFLATED, allowZip64=True)
  File "D:\Program Files\python\lib\zipfile.py", line 1204, in __init__
    self.fp = io.open(file, filemode)
PermissionError: [Errno 13] Permission denied: 'E:\\jiangman\\newtest\\test_authority\\test_data\\word.xlsx'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_12' class='none'>
    <td class='errorCase'><div class='testcase'>test_001_12</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_12' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_12'>错误</button>
    <div id='div_ft1_12' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_12' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_12'>错误</button>
    <div id='div_ft1_12' class="collapse in" align="left">
    <pre>
    
ft1_12: ----------------------------------------
https://mobileuat.utcook.com//auth-internet/resource/getResourceTags
测试场景：获取指定资源的标签列表_边界校验_"authSpace"3字符串[500]
调用的地址是:https://mobileuat.utcook.com//auth-internet/resource/getResourceTags?authSpace=k02&amp;resourceKey=ResourceKey200_001
请求参数:{'authSpace': 'k02', 'resourceKey': 'ResourceKey200_001'}
接口调用失败:{'utMsg': 'authSpace仅支持4-32个字符，允许大小写字母数字下划线！', 'utCode': 1114, 'utService': 'auth-internet-service'}:
----------------------------------------
Traceback (most recent call last):
  File "D:\Program Files\python\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\jiangman\newtest\test_authority\test_case\test_case_sit.py", line 41, in test_001
    do_excel().into_date(num + 1, (do_excel().max_column + 1), res.status_code)
  File "E:\jiangman\newtest\test_authority\common\do_excel.py", line 35, in into_date
    self.wb.save(get_common().get_excel()[0])#保存excel
  File "D:\Program Files\python\lib\site-packages\openpyxl\workbook\workbook.py", line 408, in save
    save_workbook(self, filename)
  File "D:\Program Files\python\lib\site-packages\openpyxl\writer\excel.py", line 291, in save_workbook
    archive = ZipFile(filename, 'w', ZIP_DEFLATED, allowZip64=True)
  File "D:\Program Files\python\lib\zipfile.py", line 1204, in __init__
    self.fp = io.open(file, filemode)
PermissionError: [Errno 13] Permission denied: 'E:\\jiangman\\newtest\\test_authority\\test_data\\word.xlsx'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_13' class='none'>
    <td class='errorCase'><div class='testcase'>test_001_13</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_13' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_13'>错误</button>
    <div id='div_ft1_13' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_13' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_13'>错误</button>
    <div id='div_ft1_13' class="collapse in" align="left">
    <pre>
    
ft1_13: ----------------------------------------
https://mobileuat.utcook.com//auth-internet/resource/getResourceTags
测试场景：获取指定资源的标签列表_边界校验_"authSpace"4字符串[200]
调用的地址是:https://mobileuat.utcook.com//auth-internet/resource/getResourceTags?authSpace=k022&amp;resourceKey=ResourceKey200_001
请求参数:{'authSpace': 'k022', 'resourceKey': 'ResourceKey200_001'}
接口调用成功: &lt;Response [200]&gt;
----------------------------------------
Traceback (most recent call last):
  File "D:\Program Files\python\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\jiangman\newtest\test_authority\test_case\test_case_sit.py", line 41, in test_001
    do_excel().into_date(num + 1, (do_excel().max_column + 1), res.status_code)
  File "E:\jiangman\newtest\test_authority\common\do_excel.py", line 35, in into_date
    self.wb.save(get_common().get_excel()[0])#保存excel
  File "D:\Program Files\python\lib\site-packages\openpyxl\workbook\workbook.py", line 408, in save
    save_workbook(self, filename)
  File "D:\Program Files\python\lib\site-packages\openpyxl\writer\excel.py", line 291, in save_workbook
    archive = ZipFile(filename, 'w', ZIP_DEFLATED, allowZip64=True)
  File "D:\Program Files\python\lib\zipfile.py", line 1204, in __init__
    self.fp = io.open(file, filemode)
PermissionError: [Errno 13] Permission denied: 'E:\\jiangman\\newtest\\test_authority\\test_data\\word.xlsx'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_14' class='none'>
    <td class='errorCase'><div class='testcase'>test_001_14</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_14' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_14'>错误</button>
    <div id='div_ft1_14' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_14' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_14'>错误</button>
    <div id='div_ft1_14' class="collapse in" align="left">
    <pre>
    
ft1_14: ----------------------------------------
https://mobileuat.utcook.com//auth-internet/resource/getResourceTags
测试场景：获取指定资源的标签列表_边界校验_"authSpace"32字符串[200]
调用的地址是:https://mobileuat.utcook.com//auth-internet/resource/getResourceTags?authSpace=k123456789_123456789_123456789_1&amp;resourceKey=ResourceKey200_001
请求参数:{'authSpace': 'k123456789_123456789_123456789_1', 'resourceKey': 'ResourceKey200_001'}
接口调用成功: &lt;Response [200]&gt;
----------------------------------------
Traceback (most recent call last):
  File "D:\Program Files\python\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\jiangman\newtest\test_authority\test_case\test_case_sit.py", line 41, in test_001
    do_excel().into_date(num + 1, (do_excel().max_column + 1), res.status_code)
  File "E:\jiangman\newtest\test_authority\common\do_excel.py", line 35, in into_date
    self.wb.save(get_common().get_excel()[0])#保存excel
  File "D:\Program Files\python\lib\site-packages\openpyxl\workbook\workbook.py", line 408, in save
    save_workbook(self, filename)
  File "D:\Program Files\python\lib\site-packages\openpyxl\writer\excel.py", line 291, in save_workbook
    archive = ZipFile(filename, 'w', ZIP_DEFLATED, allowZip64=True)
  File "D:\Program Files\python\lib\zipfile.py", line 1204, in __init__
    self.fp = io.open(file, filemode)
PermissionError: [Errno 13] Permission denied: 'E:\\jiangman\\newtest\\test_authority\\test_data\\word.xlsx'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_15' class='none'>
    <td class='errorCase'><div class='testcase'>test_001_15</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_15' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_15'>错误</button>
    <div id='div_ft1_15' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_15' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_15'>错误</button>
    <div id='div_ft1_15' class="collapse in" align="left">
    <pre>
    
ft1_15: ----------------------------------------
https://mobileuat.utcook.com//auth-internet/resource/getResourceTags
测试场景：获取指定资源的标签列表_边界校验_"authSpace"33字符串[500]
调用的地址是:https://mobileuat.utcook.com//auth-internet/resource/getResourceTags?authSpace=k123456789_123456789_123456789_12&amp;resourceKey=ResourceKey200_001
请求参数:{'authSpace': 'k123456789_123456789_123456789_12', 'resourceKey': 'ResourceKey200_001'}
接口调用失败:{'utMsg': 'authSpace仅支持4-32个字符，允许大小写字母数字下划线！', 'utCode': 1114, 'utService': 'auth-internet-service'}:
----------------------------------------
Traceback (most recent call last):
  File "D:\Program Files\python\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\jiangman\newtest\test_authority\test_case\test_case_sit.py", line 41, in test_001
    do_excel().into_date(num + 1, (do_excel().max_column + 1), res.status_code)
  File "E:\jiangman\newtest\test_authority\common\do_excel.py", line 35, in into_date
    self.wb.save(get_common().get_excel()[0])#保存excel
  File "D:\Program Files\python\lib\site-packages\openpyxl\workbook\workbook.py", line 408, in save
    save_workbook(self, filename)
  File "D:\Program Files\python\lib\site-packages\openpyxl\writer\excel.py", line 291, in save_workbook
    archive = ZipFile(filename, 'w', ZIP_DEFLATED, allowZip64=True)
  File "D:\Program Files\python\lib\zipfile.py", line 1204, in __init__
    self.fp = io.open(file, filemode)
PermissionError: [Errno 13] Permission denied: 'E:\\jiangman\\newtest\\test_authority\\test_data\\word.xlsx'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_16' class='none'>
    <td class='errorCase'><div class='testcase'>test_001_16</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_16' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_16'>错误</button>
    <div id='div_ft1_16' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_16' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_16'>错误</button>
    <div id='div_ft1_16' class="collapse in" align="left">
    <pre>
    
ft1_16: ----------------------------------------
https://mobileuat.utcook.com//auth-internet/resource/getResourceTags
测试场景：获取指定资源的标签列表_边界值校验_resourceKey4字符[200]
调用的地址是:https://mobileuat.utcook.com//auth-internet/resource/getResourceTags?authSpace=key_authSpace200_001&amp;resourceKey=200_
请求参数:{'authSpace': 'key_authSpace200_001', 'resourceKey': '200_'}
接口调用成功: &lt;Response [200]&gt;
----------------------------------------
Traceback (most recent call last):
  File "D:\Program Files\python\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\jiangman\newtest\test_authority\test_case\test_case_sit.py", line 41, in test_001
    do_excel().into_date(num + 1, (do_excel().max_column + 1), res.status_code)
  File "E:\jiangman\newtest\test_authority\common\do_excel.py", line 35, in into_date
    self.wb.save(get_common().get_excel()[0])#保存excel
  File "D:\Program Files\python\lib\site-packages\openpyxl\workbook\workbook.py", line 408, in save
    save_workbook(self, filename)
  File "D:\Program Files\python\lib\site-packages\openpyxl\writer\excel.py", line 291, in save_workbook
    archive = ZipFile(filename, 'w', ZIP_DEFLATED, allowZip64=True)
  File "D:\Program Files\python\lib\zipfile.py", line 1204, in __init__
    self.fp = io.open(file, filemode)
PermissionError: [Errno 13] Permission denied: 'E:\\jiangman\\newtest\\test_authority\\test_data\\word.xlsx'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_17' class='none'>
    <td class='errorCase'><div class='testcase'>test_001_17</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_17' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_17'>错误</button>
    <div id='div_ft1_17' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_17' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_17'>错误</button>
    <div id='div_ft1_17' class="collapse in" align="left">
    <pre>
    
ft1_17: ----------------------------------------
https://mobileuat.utcook.com//auth-internet/resource/getResourceTags
测试场景：获取指定资源的标签列表_边界值校验_resourceKey32字符[200]
调用的地址是:https://mobileuat.utcook.com//auth-internet/resource/getResourceTags?authSpace=key_authSpace200_001&amp;resourceKey=R123456789_123456789_123456789_1
请求参数:{'authSpace': 'key_authSpace200_001', 'resourceKey': 'R123456789_123456789_123456789_1'}
接口调用成功: &lt;Response [200]&gt;
----------------------------------------
Traceback (most recent call last):
  File "D:\Program Files\python\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\jiangman\newtest\test_authority\test_case\test_case_sit.py", line 41, in test_001
    do_excel().into_date(num + 1, (do_excel().max_column + 1), res.status_code)
  File "E:\jiangman\newtest\test_authority\common\do_excel.py", line 35, in into_date
    self.wb.save(get_common().get_excel()[0])#保存excel
  File "D:\Program Files\python\lib\site-packages\openpyxl\workbook\workbook.py", line 408, in save
    save_workbook(self, filename)
  File "D:\Program Files\python\lib\site-packages\openpyxl\writer\excel.py", line 291, in save_workbook
    archive = ZipFile(filename, 'w', ZIP_DEFLATED, allowZip64=True)
  File "D:\Program Files\python\lib\zipfile.py", line 1204, in __init__
    self.fp = io.open(file, filemode)
PermissionError: [Errno 13] Permission denied: 'E:\\jiangman\\newtest\\test_authority\\test_data\\word.xlsx'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_18' class='none'>
    <td class='errorCase'><div class='testcase'>test_001_18</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_18' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_18'>错误</button>
    <div id='div_ft1_18' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_18' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_18'>错误</button>
    <div id='div_ft1_18' class="collapse in" align="left">
    <pre>
    
ft1_18: ----------------------------------------
https://mobileuat.utcook.com//auth-internet/resource/getResourceTags
测试场景：获取指定资源的标签列表_边界值校验_resourceKey33字符[500]
调用的地址是:https://mobileuat.utcook.com//auth-internet/resource/getResourceTags?authSpace=key_authSpace200_001&amp;resourceKey=R123456789_123456789_123456789_12
请求参数:{'authSpace': 'key_authSpace200_001', 'resourceKey': 'R123456789_123456789_123456789_12'}
接口调用失败:{'utMsg': 'resourceKey仅支持4-32个字符，允许大小写字母数字下划线！', 'utCode': 1102, 'utService': 'auth-internet-service'}:
----------------------------------------
Traceback (most recent call last):
  File "D:\Program Files\python\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "E:\jiangman\newtest\test_authority\test_case\test_case_sit.py", line 41, in test_001
    do_excel().into_date(num + 1, (do_excel().max_column + 1), res.status_code)
  File "E:\jiangman\newtest\test_authority\common\do_excel.py", line 35, in into_date
    self.wb.save(get_common().get_excel()[0])#保存excel
  File "D:\Program Files\python\lib\site-packages\openpyxl\workbook\workbook.py", line 408, in save
    save_workbook(self, filename)
  File "D:\Program Files\python\lib\site-packages\openpyxl\writer\excel.py", line 291, in save_workbook
    archive = ZipFile(filename, 'w', ZIP_DEFLATED, allowZip64=True)
  File "D:\Program Files\python\lib\zipfile.py", line 1204, in __init__
    self.fp = io.open(file, filemode)
PermissionError: [Errno 13] Permission denied: 'E:\\jiangman\\newtest\\test_authority\\test_data\\word.xlsx'


    </pre>
    </div>
    </td>
</tr>

<tr id='pt1_19' class='hiddenRow'>
    <td class='passCase'><div class='testcase'>test_002</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_pt1_19' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_pt1_19'>通过</button>
    <div id='div_pt1_19' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_pt1_19' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_pt1_19'>通过</button>
    <div id='div_pt1_19' class="collapse in" align="left">
    <pre>
    
pt1_19: ----------------------------------------
----------------------------------------


    </pre>
    </div>
    </td>
</tr>

<tr id='total_row' class="text-center active">
    <td>总计</td>
    <td>19</td>
    <td>1</td>
    <td>0</td>
    <td>18</td>
    <td>通过率：5.26%</td>
</tr>
</table>

<div id='ending'>&nbsp;</div>
    <div style=" position:fixed;right:50px; bottom:30px; width:20px; height:20px;cursor:pointer">
    <a href="#"><span class="glyphicon glyphicon-eject" style = "font-size:30px;" aria-hidden="true">
    </span></a></div>
    
</body>
</html>
